src/libaktualizr/primary/eventsinterpreter.cc

```diff
@@ -115,6 +115,10 @@ void EventsInterpreter::run() {
   auto running_mode = config.uptane.running_mode;
 
   while (*events_channel >> event) {
+    if (event->variant == "DownloadProgressReport") {
+      // Process event on HMI.
+      continue;
+    }
```

Comment:
I just realized I mis-read this change. I was thinking these events were winding up in the ReportQueue, but that's not correct, right? Are there plans to also place these events in the ReportQueue?

Reply:
Correct, they are not going to the ReportQueue. We aren't planning on sending those events to the server at present, but that is something we could consider.

Reply:
Oh - i thought this might related to bug #771.

Reply:
I think we'd like to get there eventually, but it just isn't a high enough priority yet.